src/app/pages/result/result.component.ts

```diff
@@ -46,12 +51,14 @@ export class ResultComponent implements OnInit, OnDestroy {
     if (this.result) {
       this.tableData = [];
       for (let i = 1; i <= 9; i++) {
+        this.diff = this.roundTwoDecimals(this.result.originalPercentage[i] - this.result.percentage[i]);
+        this.updateScoring(this.diff);
         this.tableData.push({
           position: i,
           frequency: this.result.data[i],
           percentage: this.roundTwoDecimals(this.result.percentage[i]) + '%',
           benfordPercentage: this.roundTwoDecimals(this.result.originalPercentage[i]) + '%',
-          difference: this.roundTwoDecimals(this.result.originalPercentage[i] - this.result.percentage[i]) + '%'
+          difference: this.diff + '%',
```

Comment:
Ich würde hier das Scoring rausnehmen, da es nichts mit der Befüllung der Tabelle zu tun hat 😄 Ich glaube es macht Sinn, den Part direkt in den Benford Service (methode: calculateScore) mit aufzunehmen, dann können wir überall einen score errechnen.